src/services/DappRadarService.ts

```diff
@@ -79,7 +79,7 @@ export class DappRadarService {
             secondsAgo = (new Date().getTime() - cache.updatedAt) / 1000;
         }
 
-        if (!cache || secondsAgo > cacheValidityTime) {
+        if (!cache || secondsAgo > cacheValidityTime || !cache.data.length) {
```

Comment:
Please add a comment about what this check is looking for. The use of `||` and `>` is confusing my small brain.

Reply:
> Please add a comment about what this check is looking for. The use of `||` and `>` together is confusing my small brain.

Fixed in [this](https://github.com/AstarNetwork/astar-token-api/pull/59/commits/52ee1d5e917e0a3fbed76cec507f295b265b81e8) commit